src/main/java/com/ita/if103java/ims/service/impl/LoginServiceImpl.java

```diff
@@ -59,12 +63,15 @@ public LoginServiceImpl(UserDao userDao,
     }
 
     @Override
-    public String signIn(UserLoginDto user) {
+    public ResponseEntity signIn(UserLoginDto user) {
         try {
             authManager.authenticate(new UsernamePasswordAuthenticationToken(user.getUsername(), user.getPassword()));
             User regUser = userDao.findByEmail(user.getUsername());
             eventService.create(createEvent(regUser, LOGIN , "sign in to account."));
-            return jwtTokenProvider.createToken(user.getUsername());
+            String token = jwtTokenProvider.createToken(user.getUsername());
+            Map<Object, Object> model = new HashMap<>();
```

Comment:
Maybe 
`Map<String, String> model = new HashMap<>();`
?

Reply:
Yes, it is my inattention